src/pages/studyView/charts/ChartContainer.tsx

```
@@ -91,92 +105,154 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
+                                   title={'test'}
+                                   xAxisLabel="Months Survival"
+                                   yAxisLabel="Surviving"
+                                   totalCasesHeader="Number of Cases, Total"
+                                   statusCasesHeader="Number of Cases, Deceased"
+                                   medianMonthsHeader="Median Months Survival"
+                                   yLabelTooltip="Survival estimate"
+                                   xLabelWithEventTooltip="Time of death"
+                                   xLabelWithoutEventTooltip="Time of last observation"
+                                   showDownloadButtons={false}
+                                   showTable={false}
+                                   showLegend={false}
+                                   styleOpts={{
+                                       width: 400,
+                                       height: 380
+                                   }}
+                                   fileName="Overall_Survival"/>
+                )
+            }
+            case ChartType.SCATTER: {
+                return (
+                    <StudyViewScatterPlot
+                        width={400}
+                        height={380}
+                        onSelection={this.props.onUserSelection}
+                        data={this.props.promise.result}
+                        isSelected={d => isSelected(d, this.props.selectedSamplesMap)}
+                        isLoading={this.props.selectedSamples.isPending}
+                        selectedFill="#ff0000"
+                        unselectedFill="#0000ff"
+                        axisLabelX="Fraction of copy number altered genome"
+                        axisLabelY="# of mutations"
+                        tooltip={mutationCountVsCnaTooltip}
+                    />
+                )
+            }
+            default:
+                return null;
```

maybe throw an error here?

---

What kind of error? An element with an error message, gracefully fail but give a message in the console, or fail while compiling?